Source/.editorconfig

```diff
@@ -0,0 +1,21 @@
+﻿# EditorConfig is awesome: http://EditorConfig.org
+
+# Top-most EditorConfig file
+root = true
+
+[*]
+# 4-column space indentation
+indent_style = space
+indent_size = 4
+
+charset = utf-8
+trim_trailing_whitespace = true
+
+[*.cs]
+# Windows-style newlines
+end_of_line = crlf

```

Comment:
A while back, @adamralph commented about me missing a newline in a file. 
Ah: https://github.com/FakeItEasy/FakeItEasy/pull/566#discussion_r47742056

which was my introduction to .editorconfig. 

Do we want to take that step as well? I've no objection.